tests/testapp/tests/sync/test_controller.py

```diff
@@ -641,7 +641,7 @@ def test_deserialization_of_model_with_missing_foreignkey_referent(self):
 
         new_log.refresh_from_db()
         self.assertTrue(new_log.dirty_bit)
-        self.assertIn("missing MyUser", new_log.deserialization_error)
+        self.assertIn("my user instance with id '{}'".format(data["user_id"]), new_log.deserialization_error)
```

Comment:
Where does the space come from in between "my" and "user" here?

Reply:
Oh, I see the regex above. I think keeping it as the PascalCase class name would be more consistent with the existing stuff.

Reply:
In the dict format examples, the model name appeared with spaces, like `mastery log`. So I added a regex that turns the pascal case model name into this format. In this case, the model's name is `MyUser`.

Reply:
Ah yes, let's keep that then! 👍